storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/AbstractJdbcQueryTemplater.java

```diff
@@ -186,10 +187,10 @@ public String exportResultTableToStagingLocation(String resultTable, String stag
       throw new RuntimeException(
           String.format("Could not parse staging location: %s", stagingLocation), e);
     }
-    String stagingPath = stagingUri.getPath();
-    String exportPath =
-        String.format(
-            "%s/%s.%s", stagingPath.replaceAll("/$", ""), resultTable, EXPORT_FILE_FORMAT);
+    String bucketName = stagingUri.getHost();
+    String prefix = stagingUri.getPath();
+    String stagingPath = String.format("s3://%s%s", bucketName, prefix);
```

Comment:
the protocol should be part of the staging URI.  you shouldn't need to add it nor hardcode it.